src/App.vue

```diff
@@ -372,6 +372,7 @@ export default {
       if (this.businessId) {
         try {
           await this.fetchBusinessData() // throws on error
+          await this.fetchAddressData() // business address
```

Comment:
Since this does more than just fetching the data, I'd prefer to see it named "loadAddressData" instead.

Reply:
Or "fetchStoreAddressData"... whatever is consistent with the other functions in here.

Reply:
I changed it to "fetchStoreAddressData".